docker-compose.yaml

```diff
@@ -10,10 +10,10 @@ services:
     ports:
       - 27017:27017
 
-  fdk-fuseki-service:
-    image: eu.gcr.io/digdir-fdk-infra/fdk-fuseki-service:latest
+  fdk-sparql-service:
+    image: eu.gcr.io/digdir-fdk-infra/fdk-sparql-service:latest
     volumes:
-      - /etc/fuseki/databases/
+      - /store/app/store
```

Comment:
Ikke `/store/app/store`, men `store:/app/store` altså det skal peke til store mappe direkte i rot av prosjektet

Reply:
Ah, så du vil at jeg skal deklarere `store` som volume i tillegg. Har det så mye å si?

Jeg får uansett `No such file or directory` når jeg prøver å peke til `/app/store`, fungerer fint når jeg bruker `/store`...

Reply:
Sånn, fixnd'it ;)
58ac5ab